packages/webapp/src/shared/components/avatar/avatar.component.tsx

```diff
@@ -18,12 +18,14 @@ export const Avatar = ({ size, ...props }: AvatarProps) => {
   const avatar = currentUser?.avatar;
 
   return (
-    <Container size={size ?? 4 * themeSize.sizeUnitBase} hasImage={!isNil(avatar)} {...props}>
-      {!isNil(avatar) ? (
-        <Image src={avatar} alt={intl.formatMessage({ defaultMessage: 'user avatar', id: 'Avatar / Image alt' })} />
-      ) : (
+    <AvatarContainer {...props} className={cn(`w-[${size ? size : 40}px] h-[${size ? size : 40}px]`, props.className)}>
```

Comment:
You can set default `size` value in the line 14. This will remove double condition